INSTALL.md

```diff
@@ -0,0 +1,269 @@
+
+Other configuration settings are stored under `includes/config.php`. Everything should work out of the installation without touching those, but if  you want fax integration you will need to adjust some parameters in that file.
+
+General-purpose fax support requires customization within LibreEHR at Administration->Globals and custom/faxcover.txt; it also requires the following utilities:
+
+* faxstat and sendfax from the HylaFAX client package
+* mogrify from the ImageMagick package
+* tiff2pdf, tiffcp and tiffsplit from the libtiff-tools package
+* enscript
+
+
+##   Setting Up Access Control
+
+Since LibreEHR version `2.9.0.3`, phpGACL access controls are installed and configured automatically during LibreEHR setup.  It can be administered within LibreEHR in the admin->acl menu.  This is very powerful access control software. 
+
+Learn more about phpGACL [here](http://phpgacl.sourceforge.net/), recommend reading the phpGACL manual, the `/libreehr/Documentation/README.phpgacl` file, and the online documentation at [OpenEMR](http://www.open-emr.org) . Also recommend reading the comments in `/libreehr/library/acl.inc`.
+
+##  Upgrading
+
+**NOTE:** Most recent documentation can be found on the online documentation at [OpenEMR](http://www.open-emr.org).
+
+Be sure to back up your LibreEHR installation and database before upgrading!
+
+Upgrading LibreEHR is currently done by replacing the old libreehr directory with a newer version. And, ensure you copy your settings from the following old libreehr files to the new configuration files (we do not recommend simply copying the entire files):
+
+```
+libreehr/sites/default/config.php
+libreehr/sites/default/sqlconf.php
+```
+
+In this `sqlconf.php` file, set  `$config = 1;` (found near bottom of file within bunch of slashes)
+
+The following directories should be copied from the old version to the new version:
+```
+libreehr/sites/default/documents
+libreehr/sites/default/era
+libreehr/sites/default/edi
+libreehr/sites/default/letter_templates
+```
+
+If there are other files that you have customized, then you will also need to treat those carefully.
+
+To upgrade the database, run the `sql_upgrade.php` script from your web browser (for example: `http://libreehr.location/sql_upgrade.php`).  It will prompt you to select the old release number, and will display the SQL commands issued as the upgrade occurs.
+
+## FAQ
+
+**How can I install Apache, MySQL, and PHP on Windows?**
+
+An easy way would be to install the [XAMPP Package](https://www.apachefriends.org/index.html). Make sure to copy the LibreEHR files to the `htdocs` folder.
+
+
+**Q. How can I install Apache, MySQL, and PHP on Linux?**
+
+Follow the instructions [here](https://www.digitalocean.com/community/tutorials/how-to-install-linux-apache-mysql-php-lamp-stack-on-ubuntu-16-04)
+
+
+**Q. I'm getting `table doesn't exist` error!**
+
```

Comment:
Good job here!  Thank you for getting this straight.  Honestly, I think this should be one of the first notices in the documentation.